api/graphql/tests/test_application_rounds.py

```diff
@@ -0,0 +1,108 @@
+import json
+
+import snapshottest
+from assertpy import assert_that
+from django.contrib.auth import get_user_model
+from freezegun import freeze_time
+
+from api.graphql.tests.base import GrapheneTestCaseBase
+from applications.models import ApplicationRoundStatus
+from applications.tests.factories import (
+    ApplicationRoundFactory,
+    ApplicationRoundStatusFactory,
+)
+from permissions.models import GeneralRole, GeneralRoleChoice
+
+
+@freeze_time("2021-05-03 03:21:34")
+class ApplicationRoundQueryTestCase(GrapheneTestCaseBase, snapshottest.TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.general_admin = get_user_model().objects.create(
+            username="app_round_admin",
+            first_name="App",
+            last_name="Round",
+            email="app.round@foo.com",
+        )
+
+        GeneralRole.objects.create(
+            user=cls.general_admin,
+            role=GeneralRoleChoice.objects.get(code="admin"),
+        )
+
+        cls.application_round = ApplicationRoundFactory(
+            name="Test application round",
+            name_fi="Test application round fi",
+            name_en="Test application round en",
+            name_sv="Test application round sv",
+            criteria_fi="Criteria fi",
+            criteria_en="Criteria en",
+            criteria_sv="Criteria sv",
+        )
+        cls.application_round_status = ApplicationRoundStatusFactory(
+            application_round=cls.application_round,
+            user=cls.general_admin,
+            status=ApplicationRoundStatus.DRAFT,
+        )
+
+    def test_getting_application_rounds(self):
+        self.client.force_login(self.regular_joe)
+
+        response = self.query(
+            """
+            query {
+                applicationRounds {
+                    totalCount
+                    edges {
+                        node {
+                            id
+                            nameFi
+                            nameEn
+                            nameSv
+                            reservationUnits {
+                                id
+                            }
+                            purposes {
+                                id
+                            }
+                            serviceSector {
+                                id
+                            }
+                            status
+                            statusTimestamp
```

Comment:
this will also fail unless its some statically typed value in tests.

Reply:
This seems to be the same always and has not failed at all.

Reply:
its ok then :+1: